core/amber/src/main/scala/edu/uci/ics/texera/web/resource/auth/GoogleAuthResource.scala

```diff
@@ -14,30 +14,22 @@ import edu.uci.ics.texera.web.model.http.response.TokenIssueResponse
 import edu.uci.ics.texera.web.model.jooq.generated.enums.UserRole
 import edu.uci.ics.texera.web.model.jooq.generated.tables.daos.UserDao
 import edu.uci.ics.texera.web.model.jooq.generated.tables.pojos.User
-import edu.uci.ics.texera.web.resource.auth.GoogleAuthResource.{userDao, verifier}
-
 import java.util.Collections
 import javax.ws.rs._
-import javax.ws.rs.core.MediaType
-
-object GoogleAuthResource {
-  final private lazy val userDao = new UserDao(SqlServer.createDSLContext.configuration)
-  private val verifier =
-    new GoogleIdTokenVerifier.Builder(new NetHttpTransport, new JacksonFactory)
-      .setAudience(
-        Collections.singletonList(AmberUtils.amberConfig.getString("user-sys.googleClientId"))
-      )
-      .build()
-}
 
 @Path("/auth/google")
 class GoogleAuthResource {
+  final private lazy val userDao = new UserDao(SqlServer.createDSLContext.configuration)
+  final private lazy val clientId = AmberUtils.amberConfig.getString("user-sys.googleClientId")
   @POST
-  @Consumes(Array(MediaType.TEXT_PLAIN))
-  @Produces(Array(MediaType.APPLICATION_JSON))
```

Comment:
why do we remove those two annotations?

Reply:
They are not necessary.

Reply:
How come? they are to tell the client what format the endpoint is producing. I think we better to keep them even if it works without them. it's better to be explicit than implicit.

Reply:
Fixed.